pkg/build/openbsd.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package build
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/google/syzkaller/pkg/osutil"
+)
+
+type openbsd struct{}
+
+func (ctx openbsd) build(targetArch, vmType, kernelDir, outputDir, compiler, userspaceDir,
+	cmdlineFile, sysctlFile string, config []byte) error {
+	const kernelName = "KCOV"
```

Comment:
Do we plan to have more configs?
If we call it "KCOV", I don't see how it will develop. Will we have more? What? How users will choose?
Maybe we should just call it "syzkaller" and then enable all the different things that we need in this kernel (KCOV, KASAN, maybe something else), and then we don't need to pass this name around.

Reply:
It's just an arbitrary name. I used `KCOV` during development. Changed it to `SYZKALLER` instead, kernel config names should be capitalized.